test/test_grafana_client.py

```diff
@@ -160,7 +162,7 @@ def test_grafana_client_version(self, mock_get):
 
     def test_grafana_client_non_json_response(self):
         grafana = GrafanaApi.from_url("https://example.org/")
-        self.assertRaises(GrafanaClientError, lambda: grafana.connect())
+        self.assertRaises((GrafanaClientError, GrafanaServerError), lambda: grafana.connect())
```

Comment:
> By the way, the issue with the tests failing seems to be on line 163 of test_grafana_client.py and the behavior (?) of https://example.org/.

Indeed, example.org, on its `/api` endpoint, seems to respond with both 500 and 404 status codes now, specifically with HTTP 500 on `/api/health`.

### HTTP/1.1 404 Not Found
http HEAD https://example.org/api/foo
http HEAD https://example.org/api/hello

### HTTP/1.1 500 Internal Server Error
http HEAD https://example.org/api/bar
http HEAD https://example.org/api/world
http HEAD https://example.org/api/health

It's wierd, but c'est la vie, right?